lib/lte_link_control/lte_lc.c

```diff
@@ -671,9 +672,17 @@ static int connect_lte(bool blocking)
 	k_sem_init(&link, 0, 1);
 
 	do {
-		retry = false;
+		tries--;
 
-		if (!IS_ENABLED(CONFIG_LTE_NETWORK_DEFAULT)) {
+		err = lte_lc_func_mode_get(&current_func_mode);
+		if(err) {
```

Comment:
There should be a space after reserved keywords like `if`, most likely the CI will complain about this.

Reply:
Thanks for the heads-up! Addressed now!